stories/4-Thread.stories.tsx

```diff
@@ -73,10 +90,25 @@ export const RegularThread = () => {
       </ThreadIndent>
       <ThreadIndent
         level={2}
-        ends={[{ level: 1, onClick: () => console.log("click1") }]}
+        ends={[
+          {
+            level: 1,
+            onBeamUpClick: () => console.log("beam up click1"),
```

Comment:
Missed a couple of `console.log` here

Reply:
Yeah, I only updated the ones along-side errors. I'll fix the rest when I update this to the new format, because I'm sure a bunch of this is going to change.